From 5896fe7f8251de331754aaeb0b928d9287134931 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 8 Mar 2017 17:40:47 +0800
Subject: [PATCH 1572/5242] MLK-14392-3 arm: dts: imx7ulp: add swap property
 for touch device node

commit  700deaded4e642ceb2de976f1fcca0a56a9da485 from
https://source.codeaurora.org/external/imx/linux-imx.git

add swap property for touch device node

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry-pick from 7bf0bd714098affe65ecc0a1d80921a367b7dc1c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 66a3299..149c2ba 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -386,6 +386,7 @@
 		focaltech,keys = <139 102 158>;
 		focaltech,key-y-coord = <2000>;
 		focaltech,key-x-coords = <200 600 800>;
+		focaltech,swap-xy;
 	};
 };
 
-- 
1.7.9.5

